jpos/src/main/java/org/jpos/space/TSpace.java

```diff
@@ -18,6 +18,8 @@
 
 package org.jpos.space;
 import org.jpos.util.Loggeable;
+import org.jpos.util.NanoClock;
```

Comment:
`TSpace` instances live inside the scope of a single JVM, we don't need `NanoClock` here, it only adds overhead to a really lightweight `Space` implementation. just use `System.nanoTime()`

Reply:
True, let me try and refactor that.

Reply:
I've refactored `TSpace` to not use `Instant` at all and only use monotonic `Duration`'s.